addons/rose/addon/components/rose/form/index.stories.mdx

```diff
@@ -78,8 +81,10 @@ export const Template = (args) => ({
   as |form|
 >
   <form.input @label='Label' @helperText='Helper text' @value='Text' />
-  <form.textarea @label='Label' @helperText='Helper text' @value='Text' />
-
+  <Hds::Form::Textarea::Field @value='This is my description' as |F|>
```

Comment:
This should include disabled={{form.disabled}}, same as the below select field to show the usage